test/integration/test_usage_reports_v4.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# (C) Copyright IBM Corp. 2020.
+# (C) Copyright IBM Corp. 2023.
```

Comment:
I fear that, by re-generating this entire file, you've lost all the manual changes that were needed for the tests to run cleanly.  Were you able to run all the int tests after re-generating?

I would recommend reverting the changes to this file to get back to the original version, then generate the int tests off to the side, then copy/paste the test code blocks related to the new operations into your checked-in version.

Reply:
Also, the working examples code will need to be updated to provide working examples of the 6 new operations.